### DISCUSSION TITLE
Possible to populate migration table without running migrations?

### DISCUSSION THREAD

**Discussion by mollerhoj** [sequelize/umzug#635] [General]
This would be useful in scenarios where the database has been "migrated" manually (e.g. a snapshot of the schema has been loaded) without running the actual migrations.

Rails and Laravel has this built into their migration systems (The ability to only run migrations from a certain point, because the latest migration timestamp has been registered in the snapshot)

**Comment by mmkal** [comment]
Yes, I'd like to have this. Similar to flyway: https://documentation.red-gate.com/fd/baseline-184127456.html
